2/Dockerfile.rhel8

```diff
@@ -94,10 +94,10 @@ RUN /usr/local/bin/install-jenkins-core-plugins.sh /opt/openshift/bundle-plugins
     unlink /usr/share/man/man1/keytool.1.gz && \
     unlink /usr/share/man/man1/pack200.1.gz && \
     unlink /usr/share/man/man1/rmid.1.gz && \
-    unlink /usr/share/man/man1/rmiregistry.1.gz && \
-    unlink /usr/share/man/man1/unpack200.1.gz && \
-    chown -R 1001:0 /opt/openshift && \
-    /usr/local/bin/fix-permissions /opt/openshift && \
+    unlink /usr/share/man/man1/rmiregistry.1.gz
+RUN    unlink /usr/share/man/man1/unpack200.1.gz
+RUN    chown -R 1001:0 /opt/openshift
+RUN    /usr/local/bin/fix-permissions /opt/openshift && \
```

Comment:
Why are these being run in separate steps? This results in extra layers when the container image is built.

Reply:
I think it was being done to help weed out which commands were failing in the other pr from @akram , It may have been leftovers from troubleshooting the command, I will merge them back together and see if it works.